front/src/reducers/osrdconf/types.ts

```diff
@@ -92,6 +92,23 @@ export type PathStep = PathItemLocation & {
   isInvalid?: boolean;
 };
 
-export type StdcmPathStep = PathStep & {
-  tolerances?: { before: number; after: number };
-} & ({ isVia: true; stopType: StdcmStopTypes } | { isVia: false; arrivalType: ArrivalTimeTypes });
+export type PathItemLocationWithMetadata<T> = Extract<PathItemLocation, T> & {
+  name: string;
+  coordinates: [number, number];
+};
```

Comment:
Aren't we sure that we will have a `secondary_code` when searching it in the `Combobox` ? We could update the type or add a new one specific for that so you don't have to check everywhere if it exists.

Reply:
Maybe we can use `Extract<PathItemLocation, { uic: number }>`?